client-app/core/composables/useErrorsTranslator.ts

```diff
@@ -1,70 +1,81 @@
-import { computed } from "vue";
+import { computed, unref } from "vue";
 import { useI18n } from "vue-i18n";
-import type { Ref, ComputedRef } from "vue";
+import type {
+  IdentityErrorInfoType,
+  IdentityErrorType,
+  RegistrationErrorType,
+  ValidationErrorType,
+} from "../api/graphql/types";
+import type { ErrorType } from "../types";
```

Comment:
Lets keep it independent and not import types from graphql. Use generic in place.